server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestClusterStateAction.java

```diff
@@ -94,6 +98,7 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
             @Override
             public RestResponse buildResponse(ClusterStateResponse response, XContentBuilder builder) throws Exception {
                 builder.startObject();
+                builder.field(Fields.TIMED_OUT, response.isWaitForTimedOut());
```

Comment:
I wonder if this field should be called `wait_for_timed_out`? And I wonder if we should only include it if `wait_for_metadata_version` was set on the request?